internal/cmd/operator_list_operands.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/util/duration"
+	"sigs.k8s.io/yaml"
+
+	"github.com/operator-framework/kubectl-operator/internal/cmd/internal/log"
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+)
+
+func newOperatorListOperandsCmd(cfg *action.Configuration) *cobra.Command {
+	l := action.NewOperatorListOperands(cfg)
+	output := ""
+	validOutputs := []string{"json", "yaml"}
+
+	cmd := &cobra.Command{
+		Use:   "list-operands <operator>",
+		Short: "List operands of an installed operator",
+		Args:  cobra.ExactArgs(1),
```

Comment:
nit: could we have a `Long` section that describes some of the caveats with this command as well? 

For example, if your operands are cluster-scoped, they will be listed even if you specify a namespace. If your operator is backed by a CSV in a bad state, the list-operands call will fail since the list could be misleading. In lieu of any docs on this command the help string would be great for users.